README-offloaded-indexing.md

```diff
@@ -0,0 +1,79 @@
+### Cluster with remote indexing node, this a little manual since deploy vars are getting messy
+    $ export GIT_REL='v103rc2' && export ENCD_REL='v103rc2' && echo "$GIT_REL $ENCD_REL"
+
+    # Normal elasticsearch cluster
+    $ bin/deploy -b "$GIT_REL" --cluster-name "$ENCD_REL" --es-wait --dry-run
+        RC:   --release-candidate
+        Test: --test
+        Prod: --profile-name production --candidate
+
+    # Head frontend node: 12 app workers, 8 index workers
+    $ export ES_IP=172.31.25.24
+    $ bin/deploy -b $GIT_REL --es-ip $ES_IP --cluster-name "$ENCD_REL" --remote-indexing --primary-indexing --index-procs 8 --app-workers 12 --pg-open --do-batchupgrade no --dry-run
+        RC:   --release-candidate
+        Test: --test
+        Prod: --profile-name production --candidate
+
+    # Check environment vars
+    $ cat /etc/environment
+        * ENCD_APP_WORKERS=12
+        * ENCD_INDEX_PROCS=8
+        * ENCD_REMOTE_INDEXING=true
+
+    # ssh on and stop apache after the instance restart
+    $ sudo service apache2 stop
+
+    # Check encoded base.ini
+    [composite:indexer]
+    set queue_worker_processes = 8
+    set remote_indexing = true
+    set remote_indexing_threshold = 10001
+
+    # Check apache
+    App workers should be 12
+    indexer, vis, and region should exist
+
+    # Indexer node: 8 app workers, 16 index workers
+    $ export PG_IP=172.31.30.133
```

Comment:
Is this the frontend?

Reply:
must be.  Does the previous command list the IP or do we have to pull it off the front end.

Reply:
yes, That is a good thing to note here.